tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/parser/InvalidSyntaxParserTest.java

```diff
@@ -112,4 +112,10 @@ public void testTokenRecognition() {
         BAssertUtil.validateError(result, 0, "token recognition error at: '\\'", 3, 5);
         BAssertUtil.validateError(result, 0, "token recognition error at: '*'", 4, 15);
     }
+
+    @Test
+    public void testListenerDeclarationWithoutType() {
+        CompileResult result = BCompileUtil.compile("test-src/parser/listener-declaration-without-type.bal");
```

Comment:
Since this is a negative test case, add `-negative` part to the file name.

Reply:
done